package.json

```diff
@@ -57,19 +51,19 @@
     "mocha": "^3.2.0"
   },
   "engines": {
-    "node": ">= 0.12.0"
+    "node": ">= 4"
   },
   "ember-addon": {
     "configPath": "tests/dummy/config",
     "versionCompatibility": {
       "ember": ">=1.13"
+    },
+    "greenkeeper": {
+      "ignore": [
+        "babel-core",
+        "eslint-plugin-ember",
+        "eslint-plugin-qunit"
+      ]
     }
-  },
-  "greenkeeper": {
-    "ignore": [
-      "babel-core",
-      "eslint-plugin-ember",
-      "eslint-plugin-qunit"
-    ]
```

Comment:
@marcoow this seems to be wrong. the `greenkeeper` key should not be inside of `ember-addon`.